### DISCUSSION TITLE
See weird behavior when trying to sumulate lfp with prada2013

### DISCUSSION THREAD

**Discussion by Dhawal-Goyal** [pybamm-team/PyBaMM#2926] [Q&A]
dfn = pybamm.lithium_ion.DFN()
params = pybamm.ParameterValues('Prada2013')
experiment2 = pybamm.Experiment([
   ("Discharge at 0.1A for 30 min",  
    "Charge at 0.5A for 30 min")]
    )
sim = pybamm.Simulation(dfn, parameter_values=params, experiment=experiment2)
sol = sim.solve(calc_esoh=True)
sol.plot(["Current [A]", "Terminal voltage [V]", "Discharge capacity [A.h]", "Average negative particle concentration"])

based on above i expect the current profile to show 0.1A followed by 0.5A in the plots. but i am getting these:
![image](https://user-images.githubusercontent.com/119900847/236215566-d5f4e0c4-40c5-4d4f-add1-9a3ba0993687.png)
not sure if i am missing something here. requesting some direction/guidance

**Comment by valentinsulzer** [comment]
Which version are you using? This works fine for me on the latest version (with `calc_esoh=False`)

**Reply by Dhawal-Goyal** [reply]
I am using this pybamm==22.11.1

I used esoh=False option. mistakenly posted "esoh=True" was a typo above. I will updating to latest version
